<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="6">
     <widget class="QPushButton" name="btnClose">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>25</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>800</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="lblRdTm">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>993</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" colspan="6">
     <widget class="QScrollArea" name="scrollArea">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>999999</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>905</width>
         <height>574</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLabel" name="lblRT">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Reading time:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="btnAddGraph">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>25</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="text">
       <string>Add Graph</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="EleSyOpcUaClient::QUaBrowser" name="cwServerBrowser" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>9999999</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QUaConfigurationWidget" name="cwConfiguration" native="true">
        <property name="baseSize">
         <size>
          <width>120</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>299</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSource">
    <property name="title">
     <string>Source</string>
    </property>
    <addaction name="actionFile"/>
    <addaction name="actionServer"/>
   </widget>
   <addaction name="menuSource"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="text">
    <string>Server</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QUaConfigurationWidget</class>
   <extends>QWidget</extends>
   <header>quaconfigurationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EleSyOpcUaClient::QUaBrowser</class>
   <extends>QWidget</extends>
   <header>quabrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
